mlir/utils/jenkins/MIOpenDriver.py

```diff
@@ -378,54 +442,134 @@ def tuneMLIRKernels(configs, xdlops):
                 print("MIOpen tuning timed out")
                 outs, errs = p1.communicate()
 
+def is_xdlops_present() -> bool:
+    """This function checks whether a GPU with xdlops support is present"""
+    xdlop_supported_gpus = ['gfx908', 'gfx90a']
+    xdlop_supported_gpus_str = xdlop_supported_gpus[0]
+    for gpu in xdlop_supported_gpus[1:]:
+        xdlop_supported_gpus_str += '|' + xdlop_supported_gpus_str
+    r = subprocess.run(f"/opt/rocm/bin/rocm_agent_enumerator -t GPU | grep -E '{xdlop_supported_gpus_str}'", shell=True)
+    if r.returncode == 0:
+        return True
+    return False
+
+def add_double_dash_convention(args):
```

Comment:
Let's just make them double-dash arguments. This is an internal script that gets called from ~1 place, we can break stuff

Reply:
I was worried about me not knowing all the consumers.
Who are consumers ?

I can see Jenkinsfile  and  parametersSweeps.py being the consumers.

Reply:
Yeah, they're it. Anything using this file is in this repository.

Reply:
I had a look but I feel we should do the breaking change in a isolated PR. This PR is already seeing symptoms of scope creep.